depends/check-zziplib.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+ ls $(psp-config --psp-prefix)/lib/libz.a $(psp-config --psp-prefix)/include/zziplib.h
```

Comment:
`libz.a` should be `libzzip.a`, no?

Reply:
Oh wow, I totally overlooked that. I'll make another PR to fix it

Reply:
Fixed in #49